certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1661 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zoho Sign",
+    "description": "Zoho's Sign is a complete digital signature app that helps digitally sign business documents with complete security and reliability",
+    "version": "1.0",
+    "contact": {
+      "name": "Zoho Sign API Support",
+      "email": "support@zohosign.com",
+      "url": "https://www.zoho.com/sign/contact.html"
+    }
+  },
+  "host": "sign.zoho.com",
+  "basePath": "/api/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/x-www-form-urlencoded"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/requests/{request_id}/completioncertificate": {
+      "get": {
+        "produces": [
+          "application/pdf"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "HTTP/1.1": {
+                "description": "",
```

Comment:
@ZohoSign-integ Empty description values are not allowed. Please let us know the reason for adding this headers section with "HTTP/1.1".